fortigate-autoscale/fortigate-autoscale-settings.ts

```diff
@@ -80,7 +80,8 @@ export const FortiGateAutoscaleSettingItemDictionary: SettingItemDictionary = {
         keyName: FortiGateAutoscaleSetting.FortiGateInternalElbDns,
         description:
             'The DNS name of the elastic load balancer for the scaling ' +
```

Comment:
This line is no longer needed. It seems my previous suggestion only selected one of two lines.